Apps/Examples/Examples/All Examples/LiveDataExample.swift

```diff
@@ -0,0 +1,96 @@
+import MapboxMaps
+import Turf
+
+@objc(LiveDataExample)
+
+class LiveDataExample: UIViewController, ExampleProtocol {
+    var mapView: MapView!
+    let sourceId = "drone-source"
+    var droneTimer: Timer?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let mapInitOptions = MapInitOptions(cameraOptions: CameraOptions(zoom: 0))
+        mapView = MapView(frame: view.bounds, mapInitOptions: mapInitOptions)
+        mapView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+
+        view.addSubview(mapView)
+
+        // Add the live data layer once the map has finished loading.
+        mapView.mapboxMap.onNext(.mapLoaded) { _ in
+            self.addStyleLayer()
+        }
+    }
+
+    func addStyleLayer() {
+        // Set the data for the `GeoJSONSource`. This URL simulates paths
+        // for drones.
+        guard let url = URL(string: "https://wanderdrone.appspot.com/") else { return }
+        var source = GeoJSONSource()
+        source.data = .url(url)
+
+        var rocketLayer = SymbolLayer(id: "rocket-layer")
+        rocketLayer.source = sourceId
+        // Mapbox Streets contains an image named `rocket-15`. Use that image
+        // to represent the drone location.
+        rocketLayer.iconImage = .constant(.name("rocket-15"))
+
+        do {
+            try mapView.mapboxMap.style.addSource(source, id: sourceId)
+//            try mapView.mapboxMap.style.addLayer(rocketLayer)
+            try mapView.mapboxMap.style._addPersistentLayer(with: try rocketLayer.jsonObject(), layerPosition: nil)
```

Comment:
```suggestion
            try mapView.mapboxMap.style._addPersistentLayer(with: rocketLayer.jsonObject(), layerPosition: nil)
```

I'm 99% sure you only need to say `try` once at the beginning of the expression.